docs/documentation/js/release-info.js

```diff
@@ -0,0 +1,108 @@
+function getPageLocale() {
+  return $('.header__logo > a').first().attr('href') === '/ru/' ? 'ru' : 'en';
+}
+
+function getHue4Version(version) {
+  let result = 0
+  for (let i = 0; i < version.length; ++i)
+    result = Math.imul(31, result) + version.charCodeAt(i)
+  result = Math.abs(result)
+  return result
+}
+
+function getEditionName(edition) {
+  switch (edition) {
+    case 'fe':
+      return 'FE (Flant Edition)';
+    case 'ee':
+      return 'EE (Enterprise Edition)';
+    case 'ce':
+      return 'CE (Community Edition)';
+    default:
+      return '';
+  }
+}
+
+function getTitle() {
+  return getPageLocale() === 'ru' ? 'Дата появления версии на канале обновлений' : 'Date the version appeared on the release channel';
+}
+
+function formatDate(date) {
+  return new Intl.DateTimeFormat(getPageLocale() === 'ru' ? 'ru-RU' : 'en-US', {
+      weekday: 'long',
+      day: 'numeric',
+      month: 'long'
+  }).format(date);
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  const url = 'https://flow.deckhouse.io/releases';
+
+  const channels = ['a', 'b', 'ea', 's', 'rs'];
+  const channelCodes = {
+    "Alpha": 'a',
+    "Beta": 'b',
+    "Early Access": 'ea',
+    "Stable": 's',
+    "Rock Solid": 'rs' };
+  const editions = ['fe', 'ee', 'ce'];
+
+  const root = document.querySelector('.releases-page__table--wrap');
+  const table = document.createElement('table');
+  const thead = document.createElement('thead');
+  const trHead = document.createElement('tr');
+  const tbody = document.createElement('tbody');
+
+  fetch(url, {
```

Comment:
I suggest rewriting this promise chain into await syntax.

Reply:
There are two similar functions - showReleaseChannelStatusHorizontal and showReleaseChannelStatus. I think the first is more convenient.